ui/components/institutional/note-to-trader/note-to-trader.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Box from '../../ui/box';
+
+const NoteToTrader = (props) => {
+  const { placeholder, maxLength, onChange, noteText, labelText } = props;
+
+  return (
+    <>
+      <Box className="note-header">
+        <label htmlFor="transaction-note">{labelText}</label>
+        <p className="note-header__counter">
```

Comment:
Can we use `Text` component from `component-library` here insteading of using `p` tag?